src/todo/components/TodoItem.tsx

```diff
@@ -46,7 +48,8 @@ const TodoItem: React.FC<TodoItemProps> = ({ todo, index, setCheckedToDo, remove
     }
 
     return (
-        <>     
+        <>
```

Comment:
You can remove React Fragment wrapper due to the fact you have  the div one

Reply:
Ok, just deleted the main div with class todo-item, the rest is left